### PR TITLE
feat(gcp): add CIS checks

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Add 14 GCP checks to complete CIS.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by toniblyx — DISMISSED** [review]
Please remember to update the Description table on README.md

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ It contains hundreds of controls covering CIS, NIST 800, NIST CSF, CISA, RBI, Fe
 | Provider | Checks | Services | [Compliance Frameworks](https://docs.prowler.cloud/en/latest/tutorials/compliance/) | [Categories](https://docs.prowler.cloud/en/latest/tutorials/misc/#categories) |
 |---|---|---|---|---|
 | AWS | 283 | 55 -> `prowler aws --list-services` | 21 -> `prowler aws --list-compliance` | 5 -> `prowler aws --list-categories` |
-| GCP | 59 | 10 -> `prowler gcp --list-services` | CIS soon | 0 -> `prowler gcp --list-categories`|
+| GCP | 73 | 11 -> `prowler gcp --list-services` | 1 -> `prowler aws --list-compliance` | 0 -> `prowler gcp --list-categories`|
```
Comment (n4ch04) [inline README.md]:
```suggestion
| GCP | 73 | 11 -> `prowler gcp --list-services` | 1 -> `prowler gcp --list-compliance` | 0 -> `prowler gcp --list-categories`|
```

File: prowler/config/config.py [hunk 2]
```diff
@@ -34,7 +35,14 @@
...
     ]
-
+with os.scandir(compliance_gcp_dir) as files:
+    files = [
+        file.name
+        for file in files
+        if file.is_file()
+        and file.name.endswith(".json")
+        and available_compliance_frameworks.append(file.name.removesuffix(".json"))
+    ]
```
Comment (n4ch04) [inline prowler/config/config.py]: Could you please reutilise the above code to iterate through the compliance directories?

**Review by n4ch04 — APPROVED** [review]
🚀

**Pull request merged**